tasks/verify-conforma-konflux-ta/0.1/verify-conforma-konflux-ta.yaml

```
@@ -308,6 +308,13 @@ spec:
       command: [ec]
       args:
         - version
+    
+    - name: show-config
+      image: quay.io/enterprise-contract/ec-cli:snapshot
+      command: [jq]
+      args:
+        - ".policy.sources[].config"
+        - "json=$(params.HOMEDIR)/report-json.json"
```

Not sure about the `json=` here. Maybe this?

```suggestion
        - "$(params.HOMEDIR)/report-json.json"
```

In any case I would expect it to be the same as in the other file.

---

the error
```
Status  Error
----- Logs -----
jq: error (at /tekton/home/report-json.json:1): Cannot iterate over null (null)
```
shows /tekton/home/report-json.json is null.

---

it is not the jq, but the test case input iteself doesn't have a field of policy.sources[].config. Move test case to a different scenario and have proper snapshot defined solve this.